ClosedXML/Excel/ConditionalFormats/XLConditionalFormat.cs

```diff
@@ -110,6 +121,7 @@ public void CopyFrom(IXLConditionalFormat other)
             ReverseIconOrder = other.ReverseIconOrder;
             ShowIconOnly = other.ShowIconOnly;
             ShowBarOnly = other.ShowBarOnly;
+            StopIfTrueInternal = OpenXmlHelper.GetBooleanValueAsBool(((XLConditionalFormat)other).StopIfTrueInternal, true);
```

Comment:
This doesn't make sense. OpenXML types, e.g. BooleanValue should not be used here.